clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -824,6 +828,13 @@ private synchronized Config getResourceDescription(ConfigResource resource) {
                 }
                 return toConfigObject(clientMetricsConfigs.get(resourceName));
             }
+            case GROUP: {
+                String resourceName = resource.name();
+                if (resourceName.isEmpty()) {
+                    throw new InvalidRequestException("Empty resource name");
+                }
+                return toConfigObject(groupConfigs.get(resourceName));
```

Comment:
Should we align the behavior to return "default configs" even though the group is nonexistent?